stltovoxel.py

```diff
@@ -68,40 +43,37 @@ def exportPngs(voxels, bounding_box, outputFilePath):
 
     size = str(len(str(bounding_box[2]))+1)
     for height in range(bounding_box[2]):
-        img = Image.new('L', (bounding_box[0], bounding_box[1]), 'black')  # create a new black image
-        pixels = img.load()
-        arrayToWhiteGreyscalePixel(voxels[height], pixels)
+        print('export png %d/%d' % (height, bounding_box[2]))
+        img = Image.fromarray(voxels[height])
```

Comment:
nice

Reply:
A great time reduction has been achieved.

Reply:
https://github.com/cpederkoff/stl-to-voxel/issues/13#issuecomment-838398666
Commit: aeda356, 527 sec --> Commit: b175859, 73 sec